### PR TITLE
Feat cart screen with functionality

### PR DESCRIPTION
**Pull request by ianuragsharma**
Please check the last Commit ([feat : added cart with functionality](https://github.com/ianuragsharma/ezbuy-react/pull/3/commits/dca88b76cb78b2817fe77e2d3a4a6653da0ecf54)) 
all other commits are either have been reviewed or a refactor. 
Please ignore the comments in src/components/cartProduct/CartProduct.jsx and src/reducers/cartReducer.js 
will rectify the issue later.
Deployed link https://deploy-preview-3--idyllic-mandazi-2ce1d4.netlify.app/

### CODE REVIEW

File: src/reducers/cartReducer.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            }
+          : item
+      );
+
+    // case "INPUT_QTY":
+    //   return cartState.map((item) =>
+    //     item._id === payload.cartItemId
+    //       ? { ...item, quantity: payload.qty >= 0 ? payload.qty : 1 }
+    //       : item
+    //   );
```
Comment (Shailseen) [inline src/reducers/cartReducer.js:29]: Is this case will be needed in future?

**Review by Shailseen — COMMENTED** [review]
Can you hide counter badge on cart when no items in cart?
One thing also you can do make a page/(do changes on cart page) if your cart is empty then hide the price and give them link for route the product/homepage.

File: src/reducers/cartReducer.js (hunk 1 above)
Comment (ianuragsharma) [reply src/reducers/cartReducer.js:29]: Yes

**Comment by ianuragsharma** [comment]
> Can you hide counter badge on cart when no items in cart? One thing also you can do make a page/(do changes on cart page) if your cart is empty then hide the price and give them link for route the product/homepage.

Will do the changes .thanks for pointing out

**Review by saicharan-gouru — APPROVED** [review]
-Code looks clean and understandable.
-You can add screenshot or gif for reference while creating a the PR.
-Folder structure is so good.
-You can structure the code using code beautifiers before pushing.
-Good use of context and reducer.
-Looks good to merge.

**Pull request merged**